tools/AutomationTestFramework/ManageScripts/DeliverModuleToAutomationAccount.ps1

```diff
@@ -0,0 +1,187 @@
+﻿if (!$azureConfig) {
+    . "$PSScriptRoot\AzureConfig.ps1"
+}
+
+<#
+.SYNOPSIS
+Uploads module to Azure Storage Account container
+.PARAMETER modulePath
+Path to the directiory where module is located
+.PARAMETER moduleName
+Name of the module without extension
+.PARAMETER subscriptionName
+Subscription name of the Azure Storage Account
+.PARAMETER storageAccountName
+Azure Storage Account Name
+.PARAMETER resourceGroupName
+Resource group name of the Azure Storage Account
+.PARAMETER containerName
+Container name in the Azure Storage Account
+.NOTES
+This is the first step of the two steps upload to Azure Automation Account action
+#>
+function UploadModuleToContainer (
+     [string] $modulePath
+    ,[string] $moduleName
+    ,[string] $subscriptionName
+    ,[string] $storageAccountName
+    ,[string] $resourceGroupName
+    ,[string] $containerName) {
+
+    $subscriptionName, $resourceGroupName, $storageAccountName,  $containerName = DefaultIfNotSpecifiedSA $subscriptionName $resourceGroupName $storageAccountName  $containerName
+    Write-Host "Getting Azure storage account key..." -ForegroundColor Green
+
+    $storageAccountKey = Get-AzureRmStorageAccountKey -StorageAccountName $storageAccountName -ResourceGroupName $resourceGroupName -Verbose -ErrorAction Stop
+    $key1 = $storageAccountKey.Value[0];
+    Write-Host "Creating Azure storage account context..." -ForegroundColor Green
+    $ctx = New-AzureStorageContext $storageAccountName -StorageAccountKey $key1 -ErrorAction Stop
+    Write-Host "Uploading module to Azure storage container..." -ForegroundColor Green
+    $blobName = "$moduleName.zip"
+    $null = Set-AzureStorageBlobContent -Container $containerName  -File "$modulePath\$blobName" -Blob $blobName -Context $ctx -Force -ErrorAction Stop
+    $now = Get-Date;
+    Write-Host "Creating Azure storage blob SAS URL..." -ForegroundColor Green
```

Comment:
Write-Host -> Write-Verbose, throughout the scripts

Reply:
done.